src/webview/src/components/fbaCheckbox.js

```diff
@@ -65,6 +83,9 @@ export default function FBACheckbox(props) {
     // DataProviderEditDialog handling
     const [dpEditOpen, setDpEditOpen] = React.useState(0);
 
+    // TextFieldEditDialog handling
+    const [textFieldExitOpen, setTextFieldEditOpen] = React.useState(0);
```

Comment:
minor typo still "textFieldE**d**itOpen"

Reply:
🤦‍♂️ missed this one somehow